docker/hail-ubuntu/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:bionic-20200921
```

Comment:
Should we check whether there is a newer image available?

Reply:
It's still the latest version: https://hub.docker.com/_/ubuntu?tab=description.

Do you mean an automated check? That would be good, though I don't want to fail random PR builds if docker publishes a new ubuntu image. As we've discussed before, the real solution is a bot that automatically checks for new versions and makes PRs that update the version. For now, we'll have to periodically check for new versions and update our dependencies.

Reply:
No, I meant while we are updating this, is there a release from October.